### PR TITLE
data related ui

### PR DESCRIPTION
**Pull request by mrsaifullah52**

### CODE REVIEW

File: renderer/components/Charts/Arrows.tsx [hunk 1]
```diff
@@ -1,48 +1,45 @@
 import React from "react";
-import { fmtSize } from "../../util/minor";
+const { filesize } = require("filesize");
```
Comment (hlolli) [inline renderer/components/Charts/Arrows.tsx]: don't mix require and import statements

Comment (mrsaifullah52) [reply renderer/components/Charts/Arrows.tsx]: converted to dynamic imports

File: renderer/components/Charts/Arrows.tsx [hunk 2]
```diff
@@ -1,48 +1,50 @@
 import React from "react";
-import { fmtSize } from "../../util/minor";
+let filesize: any;
+import("filesize")
+  .then((module) => {
+    filesize = module.filesize;
+  })
+  .catch((err) => console.error("Failed to load the module", err));
```
Comment (hlolli) [inline renderer/components/Charts/Arrows.tsx]: no don't dynamically import either, are you trying to get around missing types error?

Comment (mrsaifullah52) [reply renderer/components/Charts/Arrows.tsx]: when im trying to import it normally it shows me error: 
image

Comment (hlolli) [reply renderer/components/Charts/Arrows.tsx]: enable allowSyntheticDefaultImports (by setting it to true) in tsconfig.json, and you'll be able to import it

Comment (mrsaifullah52) [reply renderer/components/Charts/Arrows.tsx]: tried but still getting the same error

Comment (mrsaifullah52) [reply renderer/components/Charts/Arrows.tsx]: github desktop
image

code
image

tsconfig
image

Comment (hlolli) [reply renderer/components/Charts/Arrows.tsx]: you may want to set this compilerOptions just in the root tsconfig, it should allow you to import cjs modules as default, let me try this...

**Review by hlolli — APPROVED** [review]

**Review by hlolli — APPROVED** [review]

**Pull request merged**